app/models/form/lettings/questions/nationality.rb

```diff
@@ -14,6 +14,8 @@ def initialize(id, hsh, page)
     "18" => { "value" => "United Kingdom" },
     "17" => { "value" => "Republic of Ireland" },
     "19" => { "value" => "European Economic Area (EEA) country, excluding Ireland" },
+    "20" => { "value" => "Afghanistan" },
+    "21" => { "value" => "Ukraine" },
```

Comment:
Are these applicable to previous forms as well?

Reply:
meaning previous years? I don't believe so. But I think to add it to previous years I would have to add them to the json right? These ruby definitions for lettings will only be active from 23/24